RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -376,44 +376,82 @@
 )
```

Comment:
```suggestion
)

def _modMapping(mapping, wpIn, wpOut):
    for m in mapping:
        m.cut = m.cut.value().replace("v1", "v3_noeveto")
        m.cut = m.cut.value().replace(wpIn, wpOut)

run2_miniAOD_UL.toModify(
    hpsPFTauDiscriminationByMVA6VLooseElectronRejection.mapping,
    func = lambda mapping: _modMapping(mapping, "WPEff99", "WPEff98")
)
```
this seems like a practical solution to go from v1 to v3, if I didn't miss some subtleties